isis/cmake/BuildDocs.cmake

```diff
@@ -83,10 +83,10 @@ function(build_upper_level)
 
   # These folders are populated inside "build_documents_folder"
 
-  # Create the main index.html file aka the home page
-  execute_process(COMMAND ${XALAN} ${XALAN_VALIDATE_OPTION} ${XALAN_PARAM_OPTION} menuPath \"${docVersion}/\" ${XALAN_OUTFILE_OPTION} ${docInstallFolder}/index.html ${XALAN_INFILE_OPTION} ${docBuildFolder}/build/homepage.xml ${XALAN_XSL_OPTION} ${docBuildFolder}/build/main.xsl)
-  # Create a second main page inside the version numbered area for when the page above gets overwritten with a new version 
-  execute_process(COMMAND ${XALAN} ${XALAN_VALIDATE_OPTION} ${XALAN_OUTFILE_OPTION} ${docInstallFolder}/${docVersion}/index.html ${XALAN_INFILE_OPTION} ${docBuildFolder}/build/homepage.xml ${XALAN_XSL_OPTION} ${docBuildFolder}/build/main.xsl)
+  # Create a redirect for the top level index file. It points to the version index file
+  execute_process(COMMAND ${XALAN} ${XALAN_VALIDATE_OPTION} ${XALAN_PARAM_OPTION} menuPath \"${docVersion}/\" ${XALAN_OUTFILE_OPTION} ${docInstallFolder}/index.html ${XALAN_INFILE_OPTION} ${docBuildFolder}/build/redirect.xml ${XALAN_XSL_OPTION} ${docBuildFolder}/build/redirect.xsl)
+  # Create the main documentaion page. This is located in the version directory 
+  execute_process(COMMAND ${XALAN} ${XALAN_VALIDATE_OPTION} ${XALAN_PARAM_OPTION} menuPath \"\" ${XALAN_OUTFILE_OPTION} ${docInstallFolder}/${docVersion}/index.html ${XALAN_INFILE_OPTION} ${docBuildFolder}/build/homepage.xml ${XALAN_XSL_OPTION} ${docBuildFolder}/build/main.xsl)
```

Comment:
I think passing through menuPath = "\" would cause the UserStart, DevStart, etc. landing pages look like absolute paths rather than realtive ones.

Reply:
You are correct, looks like I dropped that somewhere. I'm testing now to make sure I put everything back corrctly.

Reply:
I was wrong, there is a docversion already in the path for the other doc pages, so putting in an empty string starts off with 7.0.0 for the main page and menus on the doc pages are usually .. or ../..